encord/client.py

```diff
@@ -688,20 +688,22 @@ def add_classification(
 
     def create_model_row(
         self,
-        title=None,
-        description=None,
-        features=None,
-        model=None,
+        title: str,
```

Comment:
The `if title is None` checks below become  a bit annoying. For strict backward compatibility we shouldn't remove them, but I guess we could. @rad-cord do you have an opinion?